### ISSUE TITLE
flynn-host ran out of file descriptors

### ISSUE THREAD

**Issue by temujin9** [flynn/flynn#1556]
Scaling https://github.com/flynn-examples/nodejs-flynn-example up to 1100 results in "timed out waiting for scale events" at 186 processes launched. 

Salient bits from the [debug-logs](https://gist.github.com/7edb7e3f49141e38611e):
- `http: Accept error: accept tcp [::]:1113: too many open files`
- `open /proc/self/mountinfo: too many open files`

**Comment by lmars** [comment]
This is happening in CI, for example in [this build](https://ci.flynn.io/builds/20151115010657-73c9f645):

A failed job start:

```
now=2015-11-15T01:24:55+0000 fn=run job.id=abb6a4b3-8b337462-88b2-45ca-9f57-445dfa9f99c1 at=checkout status=error err="open /var/lib/docker/aufs/layers/154f4144c464e127bdc21f1a57fd831710ef2122bdd5a34c1dfc1f3785d96911: too many open files" backend=libvirt-lxc
```

Followed by failures to accept TCP connections:

```
2015/11/15 01:25:01 http: Accept error: accept tcp [::]:1113: too many open files; retrying in 5ms
t=2015-11-15T01:25:01+0000 lvl=info msg="request started" component=host req_id=c4bbb1a5-9fd4-4bb4-a933-060bfe78fe0d method=GET path=/host/jobs client_ip=10.69.6.4
2015/11/15 01:25:01 http: Accept error: accept tcp [::]:1113: too many open files; retrying in 10ms
t=2015-11-15T01:25:01+0000 lvl=info msg="request completed" component=host req_id=c4bbb1a5-9fd4-4bb4-a933-060bfe78fe0d status=200 duration=33.910747ms
2015/11/15 01:25:01 http: Accept error: accept tcp [::]:1113: too many open files; retrying in 20ms
2015/11/15 01:25:01 http: Accept error: accept tcp [::]:1113: too many open files; retrying in 40ms
2015/11/15 01:25:01 http: Accept error: accept tcp [::]:1113: too many open files; retrying in 80ms
2015/11/15 01:25:01 http: Accept error: accept tcp [::]:1113: too many open files; retrying in 160ms
2015/11/15 01:25:02 http: Accept error: accept tcp [::]:1113: too many open files; retrying in 320ms
2015/11/15 01:25:02 http: Accept error: accept tcp [::]:1113: too many open files; retrying in 640ms
2015/11/15 01:25:03 http: Accept error: accept tcp [::]:1113: too many open files; retrying in 1s
2015/11/15 01:25:04 http: Accept error: accept tcp [::]:1113: too many open files; retrying in 1s
```

I am digging more into the cause in this specific case.

**Comment by titanous** [comment]
The leak is fixed in #2293, though running "too many" containers will still exhaust the default fd limit.

**Comment by temujin9** [comment]
This appears fixed; I will reopen a new issue if similar occurs in future.
